FiveCalls/FiveCalls/Issue.swift

```diff
@@ -42,6 +43,6 @@ extension Issue : JSONSerializable {
         
         let contacts = contactsJSON.flatMap({ Contact(dictionary: $0) })
         
-        self.init(id: id, name: name, reason: reason, script: script, contacts: contacts)
+        self.init(id: id, name: name, reason: reason, script: script, contacts: contacts, madeCalls: false)
```

Comment:
Yeah I think that's the way to do it, very simple local app data that references the JSON Structs.